src/lwc/bdiObjectMappings/bdiObjectMappings.html

```diff
@@ -33,6 +32,26 @@ <h1 class="slds-text-heading_medium slds-text-heading_medium_extension slds-m-bo
                 <template if:true={shouldRender}>
                     <h2 class="slds-text-heading_small slds-text-heading_small_extension slds-m-bottom_medium">{customLabels.bdiOMUIObjectGroupsTitle}</h2>
 
+                    <template if:true={hasBrokenMetadataReferences}>
+                        <div class="slds-size_1-of-1">
+                            <c-util-page-level-message title='Check Field Mappings'
```

Comment:
Are we adding custom labels in a separate work items for the title and subtitle here?